getting_started.md

```diff
@@ -0,0 +1,166 @@
+
+![Scuttlebutt Logo](assets/hermies-300x300.png)
+
+Ok, you've been hearing about [Scuttlebutt](https://scuttlebutt.nz)  and decided
+that "Yes, I do want to join an amazing social network with lots of good people
+that no company can control and also happens to also work offline. 
+
+Here's a quick overview with the basics you need to know.
+
+NOTE: If you have no time, and just need to get up and running ASAP, there's a
+[Quick Start](README.md#quick-start) in the introduction. 
+
+
+First off Scuttlebutt is a protocol on which many different kinds of apps can be
+built. 
+
+As for the social network, there are many clients, just like there are many
+Twitter clients. It doesn't really matter which one you use. They're all talking
+on the same network. 
+
+Patchwork is the most polished client and it doesn't require you to know any
+geeky stuff. The only caveat is that it just wants to run on one computer, and
+your identity is tied to the files on the computer you install it on. There are
+ways around this but it's beyond the scope of this.
+
+![screenshot of Patchwork](assets/patchwork.jpg)
+
+
+## Step 0 If you haven't done so already go to [the Scuttlebutt Home
+Page](README.md) and watch the video. Then come back. 
```

Comment:
formatting?

Reply:
could be totally fine, i see that you're wrapping lines at a max character length. happens again a couple times later below.

Reply:
nope. i reformatted the page before saving and didn't realize it'd do that to the headings with no line break between them and the 1st line. it'd screw up any md renderer and make it all a heading.